README.md

```diff
@@ -40,11 +99,14 @@ With these variables you can define a few variables for the VPC that will be cre
       type        = string
       default     = "10.0.0.0/16"
     }
+```
 
-### EKS Worker Nodes configration
+#### EKS Worker Nodes configuration
+<a id="markdown-eks-worker-nodes-config" name="eks-worker-nodes-config"></a>
 
-With these variables you can configure the worker nodes pool for the EKS cluster:
+You can configure the EKS worker node pool with the following variables:
 
+```
     variable "desired_number_of_nodes" {
```

Comment:
I am still a bit sceptical about adding the Terraform variable definition here. Now you have to places in the docs where this is documented and the ones which are added into the text (opposed to the generated table) need to be kept manually up to date.